app/views/users/settings/_sidebar.html.erb

```diff
@@ -63,7 +63,12 @@
        <li class="<%= 'active' if on_settings_team_page? %>" >
          <span class="tree-link line-wrap no-indent">
            <% if on_settings_team_page? %>
```

Comment:
Do we still need this `if... else` sentence, or can we merge this together?

Reply:
I have no idea.

Reply:
You can give it a try :smiley_cat:

Reply:
it has to be like this